src/pages/documentation/media-for-edge-network/tabs/api-reference.md

```diff
@@ -0,0 +1,2117 @@
+                                                            length:60 
+                                                        streamType:AEPEdgeMediaStreamType.VOD
+                                                         mediaType:AEPEdgeMediaTypeVideo];
+```
+
+<Variant platform="android" api="create-ad-break-object" repeat="8"/>
+
+#### Java
+
+**Syntax**
+
+```java
+public static HashMap<String, Object> createAdBreakObject(String name, int position, int startTime);
+```
+
+**Example**
+
+```java
+HashMap<String, Object> adBreakObject = Media.createAdBreakObject("adbreak-name", 1, 0);
+```
+
+#### Kotlin
+
+**Example**
+
+```java
+val adBreakObject = Media.createAdBreakObject("adbreak-name", 1, 0)
+```
+
+<Variant platform="ios" api="create-ad-break-object" repeat="11"/>
+
+Returns a map that contains information about the ad break.
+
+#### Swift
+
+**Syntax**
+
+```swift
+static func createAdBreakObjectWith(name: String,
+                                position: Int, 
+                                startTime: Int) -> [String: Any]?
+```
+
+**Example**
+
+```swift
+let adBreakObject = Media.createAdBreakObjectWith(name: "adbreak-name", 
+                                              position: 1, 
+                                             startTime: 0)
+```
+
+#### Objective-C
+
+**Syntax**
+
+```objectivec
++ (NSDictionary  <NSString *, id> * _Nullable) createAdBreakObjectWith:(NSString * _Nonnull)position:(NSInteger) startTime:(NSInteger)
+```
+
+**Example**
+
+```objectivec
+NSDictionary *adBreakObject = [AEPMobileEdgeMedia createAdBreakObjectWith:@"adbreak-name" 
+                                                             position:1 
+                                                            startTime:0];
+```
+
+<Variant platform="android" api="create-ad-object" repeat="8"/>
+
+#### Java
+
+**Syntax**
+
+```java
+public static HashMap<String, Object> createAdObject(String name, String adId, int position, int length);
+```
+
+**Example**
+
+```java
+HashMap<String, Object> adInfo = Media.createAdObject("ad-name", "ad-id", 1, 15);
+```
+
+#### Kotlin
+
+**Example**
+
+```java
+val adInfo = Media.createAdObject("ad-name", "ad-id", 1, 15)
+```
+
+<Variant platform="ios" api="create-ad-object" repeat="11"/>
+
+Returns a map that contains information about the ad.
+
+#### Swift
+
+**Syntax**
+
+```swift
+static func createAdObjectWith(name: String, 
+                                 id: String, 
+                           position: Int, 
+                             length: Int) -> [String: Any]?
+```
+
+**Example**
+
+```swift
+let adObject = Media.createObjectWith(name: "ad-name", 
+                                        id: "ad-id", 
+                                  position: 0, 
+                                    length: 30)
+```
+
+#### Objective-C
+
+**Syntax**
+
+```objectivec
++ (NSDictionary  <NSString *, id> * _Nullable) createAdObjectWith: (NSString * _Nonnull
```

Comment:
In `createAdObjectWith: (NSString * _Nonnull`, remove the extra space after the colon and add the closing parentheses.